pkg/ovs/ovn.go

```diff
@@ -71,16 +69,38 @@ func NewLegacyClient(timeout int, ovnSbAddr, clusterRouter, clusterTcpLoadBalanc
 	}
 }
 
-// TODO: support sb/ic-nb client
-func NewOvnClient(ovnNbAddr string, ovnNbTimeout int, nodeSwitchCIDR string) (*ovnClient, error) {
-	nbClient, err := ovsclient.NewNbClient(ovnNbAddr)
+func NewOvnNbClient(ovnNbAddr string, ovnNbTimeout int, nodeSwitchCIDR string) (*ovnNbClient, error) {
```

Comment:
```suggestion
func NewOvnNbClient(ovnNbAddr string, ovnNbTimeout int) (*ovnNbClient, error) {
```

Reply:
My bad. `nodeSwitchCIDR` should be removed from `NewOvnSbClient ` instead of `NewOvnNbClient()`.